crates/sui-graphql-rpc/Cargo.toml

```diff
@@ -28,6 +28,7 @@ prometheus.workspace = true
 rand.workspace = true # todo: cleanup test only deps
 regex.workspace = true
 reqwest.workspace = true
+serial_test = { version = "2.0.0"}
```

Comment:
@oxade, is there a reason this isn't a `workspace` dependency? It's the only one.

Reply:
It was just convenient to plop it in, but sure addressed here https://github.com/MystenLabs/sui/pull/14489